app/assets/stylesheets/modules/forms.scss

```diff
@@ -36,14 +37,35 @@
 }
 
 .form-group h2,
+.form-group .success-page .requested-by,
 .searchworks-item {
   font-family: 'dejavu_sansextralight', $font-family-base;
-  margin: 0 0 1em;
+  line-height: 1.6em;
+  margin: 0 0 0.5em;
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
 }
 
+.required:after {
+  content: 'Required';
+  margin-left: 5px;
+}
```

Comment:
`.required:after` is already defined above https://github.com/sul-dlss/sul-requests/pull/300/files#diff-22681f8e52bfcda88e3b3a6e9a7982a0L21